zapisy/apps/users/views.py

```diff
@@ -159,9 +159,11 @@ def my_profile(request):
     consultations).
     """
     semester = Semester.get_upcoming_semester()
+    is_ext_contractor = is_external_contractor(request.user)
```

Comment:
```suggestion
    is_external_contractor = is_external_contractor(request.user)
```
Jeszcze nie jest tak źle z długością naszych identyfikatorów / wierszy kodu.